lib/rdoc/generator/template/darkfish/css/rdoc.css

```diff
@@ -265,6 +266,7 @@ main {
   font-size: 16px;
   line-height: 1.6;
   color: var(--text-color);
+  box-sizing: border-box;
```

Comment:
This could probably go on the `html` selector if it's not too disruptive. I think most sites use it because it's more intuitive.

Reply:
Moving it to `body` actually would make the mobile view zoom in again but I'm not sure why. So I'll leave it here for now but I'd appreciate a later PR to put it to a better place.